src/components/app.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import {Link} from 'react-router';
+import PropTypes from 'prop-types';
+
+import { filterList, deletePerson } from '../actions';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.searchName = this.searchName.bind(this);
+  }
+  searchName() {
+    this.props.filterList(this.searchInput.value);
+  }
+  delete(id) {
+    this.props.deletePerson(id);
+  }
+  render() {
+    return (
+      <div className="app">
+        <div className="app-title">Contact-list</div>
+        <input
+          className="search-field"
+          placeholder="..."
+          ref={(input) => { this.searchInput = input }}
+          onChange={this.searchName}
+        />
+        <Link to={'/add-contact'}>
+          <button>add contact</button>
+        </Link>
+        <div className="grid">
+          {this.props.person.map( (elem) => {
```

Comment:
https://dab1nmslvvntp.cloudfront.net/wp-content/uploads/2016/02/1454335923es6-arrow-functions-new-fat-and-concise-syntax-in-js06-object-literal-syntax.png